uploader/config.txt

```diff
@@ -1,3 +1,4 @@
+GRANT_TYPE="client_credentials"
```

Comment:
Can we leave this empty and update the value in the [sample_config](https://github.com/onaio/fhir-tooling/blob/main/uploader/sample_config.txt) instead ?
Maybe also update the [readme](https://github.com/onaio/fhir-tooling/blob/main/uploader/README.md) to show the options